server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 from pandas.core.dtypes.dtypes import CategoricalDtype
-import scanpy as sc
+import anndata
```

Comment:
Why the switch from scanpy to anndata? I think it's fine (it actually makes it more clear that we're just using the data structure), just wondering what the thought process was?

Reply:
it imports in 1/2 of the time - so we get faster startup.

Reply:
plus, we aren't importing a bunch of stuff into memory that is unused.